### ISSUE TITLE
Accessibility: Section lacks heading

### ISSUE THREAD

**Issue by abhipanda** [SSENSE/vue-carousel#388]
## Bug Report

Incorrect use of section tag causes HTML syntax to be invalid. The section tag used MUST have a heading.

Fix is to convert the section tag to a div.

**Comment by quinnlangille** [comment]
Hey @abhipanda, thanks for submitting the fix! I'll review asap

**Comment by abhipanda** [comment]
Thanks @quinnlangille
